spec/controllers/tasks_controller_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
+
+RSpec.describe TasksController, type: :controller do
+  let!(:user) { FactoryBot.create(:user) }
+  let!(:assignment) { FactoryBot.create(:assignment) }
+  let!(:task) { FactoryBot.create(:task, creator: user, user_ids: user.id) }
+  before { sign_in(user) }
+
+  describe 'GET #index' do
+    let(:task2) { FactoryBot.create(:task, creator: user, user_ids: user.id) }
+
```

Comment:
` before { get :index, params: { user_id: user.id } }`

DRY your code by adding this line here